README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/twilio-labs/function-templates.svg?branch=master)](https://travis-ci.org/twilio-labs/function-templates)
 
-[Twilio Functions](https://www.twilio.com/functions) are a serverless environment to build and run Twilio applications so you can get to production faster. You provide the Node.js code to perform the task you need and Twilio runs it. You can read [more about Twilio Functions and how to use them in the introductory blog post](https://www.twilio.com/blog/2017/05/introducing-twilio-functions.html).
+[Twilio Functions](https://www.twilio.com/functions) are a serverless environment to build and run Twilio applications so you can get to production faster. You provide the Node.js code to perform the task you need and Twilio runs it.Runtime consists of a set of tools including a [CLI](https://www.twilio.com/docs/twilio-cli/quickstart), [helper libraries](https://www.twilio.com/docs/libraries),
+[API keys](https://www.twilio.com/docs/iam/keys/api-key), debugging tools(https://www.twilio.com/docs/usage/troubleshooting/debugging-your-application), a visual workflow designer called Studio, and a serverless hosting environment for Node.js code and static assets: Twilio Functions & Assets. You can read [more about Twilio Functions and how to use them in the introductory blog post](https://www.twilio.com/blog/2017/05/introducing-twilio-functions.html).
```

Comment:
This is unnecessary, this project is concerned with Twilio Functions and Assets only and doesn't need a paragraph about Runtime [copied from the docs](https://www.twilio.com/docs/runtime).